src/sentry/static/sentry/app/components/group/seenInfo.jsx

```diff
@@ -75,26 +43,54 @@ const SeenInfo = createReactClass({
     return `/${orgId}/${projectId}/settings/release-tracking/`;
   },
 
+  getTooltipTitle() {
+    let {date, dateGlobal, environment, title} = this.props;
+
+    return componentToString(
+      <div style={{width: 170}}>
+        <div className="time-label">{title}</div>
+        <dl className="flat">
+          {environment && [
+            <dt key="0">{toTitleCase(environment)}</dt>,
+            <dd key="0.1">
+              <TimeSince date={date} />
+              <br />
+            </dd>,
+          ]}
+          <dt key="1">{t('Globally:')}</dt>
+          <dd key="1.1">
+            <TimeSince date={dateGlobal} />
+            <br />
+          </dd>
+        </dl>
+      </div>
+    );
+  },
+
   render() {
     let {date, dateGlobal, environment, release, orgId, projectId} = this.props;
     return (
       <dl className="seen-info">
         <dt key={0}>{t('When')}:</dt>
         {date ? (
           <dd key={1}>
-            <span className="tip">
-              <TimeSince date={date} />
-            </span>
+            <Tooltip title={this.getTooltipTitle} tooltipOptions={{html: true}}>
+              <span>
```

Comment:
Is `<span>` still needed?

Reply:
Sadly yes because `TimeSince` has a title property, and Tooltip wants to add the title property to the child element